Library/ViewModels/RewardsCollectionViewModelTests.swift

```diff
@@ -157,6 +163,131 @@ final class RewardsCollectionViewModelTests: TestCase {
     XCTAssertEqual(self.vm.outputs.selectedReward(), project.rewards[endIndex - 1])
   }
 
+  func testGoToManagePledge_featureNativeCheckoutPledgeView_Enabled() {
+    let config = .template
+      |> Config.lens.features .~ [Feature.nativeCheckoutPledgeView.rawValue: true]
```

Comment:
How are we going to proceed from here taking `nativeCheckoutV1` into account? Should we test for both of those flags or is `nativeCheckoutPledgeView` sufficient?

|> \.abExperiments .~ [Experiment.Name.nativeCheckoutV1.rawValue: "experimental"]

Reply:
It's not possible to go to `ManagePledgeViewController` if the `nativeCheckoutV1` is disabled, because the only place we are presenting the viewController from the Rewards Carousel.

Reply:
The `ManagePledgeViewController` is only accessible if `nativeCheckoutV1` is experimental, because we're presenting from the Rewards Carousel. That said, `nativeCheckoutPledgeView` should be sufficient.